tests/wpt/metadata/html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/window-iframe-messagechannel-success.html.ini

```diff
@@ -1,5 +1,4 @@
 [window-iframe-messagechannel-success.html]
-  expected: TIMEOUT
   [postMessaging to a same-origin iframe via MessageChannel allows them to see each others' modifications]
```

Comment:
Note: this tests expects to send a port as a message via structured clone(not as the transfer argument, as the actual message)